doc/nrf/ug_thread_tools.rst

```diff
@@ -196,6 +167,17 @@ However, this approach requires you to download the OpenThread Border Router rep
 To set up and configure the OpenThread Border Router, follow the official `OpenThread Border Router Codelab tutorial`_ on the OpenThread documentation portal.
 Omit the *Build and flash RCP firmware* section, because this section duplicates the steps performed in the previous section.
 
+.. note::
+
+   OTBR is developing every day and it may happen that the newest version is incompatible with the specific version of NCS.
+   To make sure that everything is working correctly, use a specific commit of OTBR that has been tested with the corresponding version of NCS.
+
+   +--------------------------+--------------------+
+   | OpenThread Border Router |        NCS         |
+   +==========================+====================+
+   |        e149a60eb         |       v1.7         |
```

Comment:
```suggestion
   |        e149a60e          |       v1.7         |
```
Maybe use an even number of characters for the SHA?

Reply:
Why do you want to use event number?

Reply:
Never mind, just realized that GitHub itself is using 7 chars as abbreviation.